frontend/src/metabase/dashboard/containers/DashCardCardParameterMapper.jsx

```diff
@@ -172,6 +172,8 @@ export default class DashCardCardParameterMapper extends Component {
                 <span className="text-centered mr1">
                   {disabled
                     ? t`No valid fields`
+                    : selected && selected.sectionName == null
+                    ? selected.name // for native question variables we just display the name
```

Comment:
This ternary feels like it's getting a bit out of hand.

Reply:
Good point. I guess it's doing two things: 1. Are we showing a name? 2. What name are we showing?

I'll extract the second one to elsewhere.